### DISCUSSION TITLE
FATAL:  role "-d" does not exist

### DISCUSSION THREAD

**Discussion by TheOnlyWayUp** [umami-software/umami#3080] [Q&A]
The database container has this error often
![image](https://github.com/user-attachments/assets/0c4194ef-a7ce-4714-ba16-618c5e32ba01)

The service works as expected, is this something I should be alarmed about?

**Comment by mikecao** [comment]
Looks unrelated to Umami, likely something in your db config.

**Reply by TheOnlyWayUp** [reply]
I'm using the [Docker Compose](https://raw.githubusercontent.com/umami-software/umami/e1cd8eac8351b39171b3a08ab69b659464ffc40a/docker-compose.yml) in the root of this project, no other service uses this db.

Is there anything I can run to narrow it down further?
